torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3394,34 +3400,59 @@ def error_inputs_embedding(op_info, device, **kwargs):
 
 def error_inputs_multinomial(op_info, device, **kwargs):
     x = torch.empty(1, 2, 3, dtype=torch.double, device=device)
-    yield ErrorInput(SampleInput(x, args=(2,)), error_type=RuntimeError,
+    yield ErrorInput(SampleInput(x, args=(2,)),
```

Comment:
Why remove the error type? Is the type of the error inconsistent across device types?

Reply:
OH it's because it's the default. Sure.

Reply:
Yes! :)